src/Rx.ts

```diff
@@ -20,6 +20,7 @@ import './add/observable/fromArray';
 import './add/observable/fromEvent';
 import './add/observable/fromEventPattern';
 import './add/observable/fromPromise';
+import './add/observable/generate';
```

Comment:
if this is going under `Rx`, it may need to be also under `kitchenSink` as well. (`kitchSink` contains all of `Rx` export)